spread/qemu.go

```diff
@@ -10,6 +10,12 @@ import (
 	"golang.org/x/net/context"
 )
 
+var (
+	ovmfDefaultPath = "/usr/share/OVMF/OVMF_CODE.fd"
```

Comment:
Having a default for ubuntu/debian sounds good. Let's revert it and look at `.spread/qemu/bios/<bios-name>.img` (e.g. uefi.img) first and if nothing is there we use the default ovmfDefaultPath above.